website/docs/cloud-docs/workspaces/variables/index.mdx

```diff
@@ -74,6 +74,16 @@ Each environment and Terraform variable can have one of the following scopes:
 | Project-Scoped Variable Set   | Automatically applied to all current and future workspaces within a project.       | [Create Variable Sets](/terraform/cloud-docs/workspaces/variables/managing-variables#variable-sets) and [Variable Sets API](/terraform/cloud-docs/api-docs/variable-sets)                                                                                                                                                                              |
 | Global Variable Set           | Automatically applied to all current and future workspaces within an organization. | [Create Variable Sets](/terraform/cloud-docs/workspaces/variables/managing-variables#variable-sets) and [Variable Sets API](/terraform/cloud-docs/api-docs/variable-sets)                                                                                                                                                                              |
 
+## Variable Set Ownership
+
+Variable set ownership determines whether the variable set is managed at the organization level or the project level.
+
+Use organization-owned variable sets to share variables across multiple projects. Because they can be applied to workspaces in any project in the organization, managing organization-owned variable sets requires a higher level of permissions.
```

Comment:
Love the clear recommendation here! Suggestion for flow and active voice:
```suggestion
Use organization-owned variable sets to share variables across multiple projects. Managing organization-owned variable sets [requires a higher permission level](/terraform/cloud-docs/workspaces/variables/managing-variables#permissions) because you can apply these sets to any project in your organization.
```